tests/hypervisor/test_hyperv.py

```diff
@@ -634,3 +637,96 @@ def test_exclude_host(self, virtwho, function_hypervisor, hypervisor_data):
                 and result["thread"] == 1
                 and hostname not in str(result["mappings"])
             )
+    
+    @pytest.mark.tier3
+    def test_biosguid_null(self, virtwho, function_hypervisor, hypervisor_data, register_data):
+        """Test if the biosguid is null, whether virt-who can send the guest info to server.
+        
+        :title: virt-who: hyperv: test biosguid null
+        :id: 4de4fed0-97c2-4146-9540-2223386297c4
+        :caseimportance: High
+        :tags: hypervisor,hyperv,tier3
+        :customerscenario: false
+        :upstream: no
+        :steps:
+            1. Run virt-who with normal guid.
+            2. Download the New-VMBIOSGUID.ps1 script from the server.
+            3. Import the New-VMBIOSGUID.ps1 script.
+            4. Change the biosguid to null.
+            5. Run virt-who again.
+            6. Change the biosguid to the original value.
+        :expectedresults:
+            1. Succeeded to run the virt-who, can find the guest info in the server.
+            2. Succeeded to download the New-VMBIOSGUID.ps1 script.
+            3. Succeeded to import the New-VMBIOSGUID.ps1 script.
+            4. Succeeded to change the biosguid to null.
+            5. Succeeded to run the virt-who, cannot find the guest info in the server.
+        """
+        
+        hyperv = HypervCLI(
+            server = hypervisor_data["hypervisor_server"],
+            ssh_user = hypervisor_data["hypervisor_username"],
+            ssh_pwd = hypervisor_data["hypervisor_password"],
+        )
+        
+        result = virtwho.run_service()
+        assert (
+            result["error"] == 0
+            and result["send"] == 1
+            and result["thread"] == 1
+        )
+        
+        get_guid_cmd = r"PowerShell (gwmi -Namespace Root\Virtualization\V2 -ClassName Msvm_VirtualSystemSettingData).BiosGUID"
+        ret, origin_guid = hyperv.ssh.runcmd(get_guid_cmd)
+        assert origin_guid != "" and ret == 0
+
+        create_function_cmd = r"PowerShell (Invoke-WebRequest http://10.73.131.85/ci/hyperv/New-VMBIOSGUID.ps1 -OutFile ./New-VMBIOSGUID.ps1)"
+        ret, _ = hyperv.ssh.runcmd(create_function_cmd)
+        assert ret == 0
+        
+        import_module_cmd = r"Import-Module ./New-VMBIOSGUID.ps1 -Force"
+        set_ignore_verfiy_cmd = r"$ConfirmPreference = 'None'"
+        change_guid_cmd = 'PowerShell -Command "{}; {}; New-VMBIOSGUID -VM {} -NewID 00000000-0000-0000-0000-000000000000"'.format(set_ignore_verfiy_cmd,
+                                                                                                                                      import_module_cmd,
+                                                                                                                                      hypervisor_data['guest_name'])
```

Comment:
Move these functions to [hypervisor-builder](https://github.com/VirtwhoQE/hypervisor-builder/) project